ha_vpn_gateway_gcp_to_gcp/main.tf

```diff
@@ -69,6 +69,7 @@ resource "google_compute_subnetwork" "network2_subnet2" {
 
 resource "google_compute_router" "router1" {
```

Comment:
Can you make your changes in https://github.com/terraform-google-modules/terraform-docs-samples/tree/main/vpn?

We are in the process of a repo reorg.

Reply:
Ah! Yes, sorry -- I didn't realize, yes. Will do this.